account_analytic_required/tests/test_account_analytic_required.py

```diff
@@ -188,13 +161,3 @@ def test_posted_ok(self):
         move = line.move_id
         move.action_post()
         self.assertEqual(move.state, "posted")
-
-    def test_exception_policy_for_analytic_distribution(self):
```

Comment:
Any reason to drop this test? Isn't it still in the scope of the module?

Reply:
Analytic Distribution is Analytic, no need to make this test anymore.